udata/harvest/api.py

```diff
@@ -241,6 +243,8 @@ def put(self, ident):
     @api.doc('delete_harvest_source')
     @api.marshal_with(source_fields)
     def delete(self, ident):
+        source = actions.get_source(ident)
```

Comment:
I would add a little type hint here so that reader knows what's the type without having to inspect `faker.url()` method, but might be because I'm a type hint nazi :)